Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule_test.py

```diff
@@ -2287,3 +2273,22 @@ def test_create_x509_certificate_object():
         "issuer": "C=ZA, ST=Western Cape, L=Cape Town, O=Thawte Consulting cc, OU=Certification Services Division,"
         " CN=Thawte Server CA/emailAddress=server-certs@thawte.com",
     }
+
+
+@pytest.mark.parametrize(
+    "pattern, value",
+    [
+        ("[domain-name:value = 'www.example.com']", 'www.example.com'),
```

Comment:
Lets add some use case, we have url, domain, MD5,
[domain-name:value = 'www.example.com']
[file:hashes.'MD5' = 'xxx']

Reply:
Done